visualization/app/codeCharta/ui/attributeSideBar/attributeSideBar.component.ts

```diff
@@ -40,12 +40,14 @@ export class AttributeSideBarController
 		primaryMetricKeys: PrimaryMetrics
 		secondaryMetricKeys: string[]
 		isSideBarVisible: boolean
+		fileCount: number
 	} = {
 		node: null,
 		fileName: null,
 		primaryMetricKeys: { node: {}, edge: {} } as PrimaryMetrics,
 		secondaryMetricKeys: null,
-		isSideBarVisible: null
+		isSideBarVisible: null,
+		fileCount: 0
```

Comment:
We could improve the name a little bit. Suggestion: `packageFileCount`. Feel free to update the name.